src/main/java/edu/uw/cse/testbayes/model/Bayes.java

```diff
@@ -101,42 +103,11 @@ public Probability getFailCondProb(String s1, String s2) {
     return tot;
   }
 
-  // Uses the execution map provided to produce Conditional probability that a test will pass based on other tests passing.
-  private Map<String, Map<String, Probability>> buildPasscond(Map<String, Map<String, Double>> testExecs) {
-    Map<String, Map<String, Probability>> passcond = new HashMap<String, Map<String, Probability>>();
-    Set<String> executions = testExecs.keySet();
-    for (String execution : executions) {
-      Map<String, Double> tests = testExecs.get(execution);
-      Set<String> thisexec = tests.keySet();
-      for (String test1 : thisexec) {
-        for (String test2 : thisexec) {
-          Double time1 = tests.get(test1);
-          Double time2 = tests.get(test2);
-          if (!passcond.containsKey(test1)) {
-            passcond.put(test1, new HashMap<>());
-          }
-          Map<String, Probability> conds1 = passcond.get(test1);
-          if (!conds1.containsKey(test2)) {
-            conds1.put(test2, new Probability(NUMERATOR, DENOMINATOR));
-          }
-          Probability prob = conds1.get(test2);
-          if (time1 > 0) {
-            if (time2 > 0) {
-              prob.addNumerator(1);
-            }
-            prob.addDenominator(1);
-            conds1.put(test2, prob);
-            passcond.put(test1, conds1);
-          }
-        }
-      }
-    }
-    return passcond;
-  }
 
-  // Uses the execution map provided to produce Conditional probability that a test will pass based on other tests failing.
-  private Map<String, Map<String, Probability>> buildFailcond(Map<String, Map<String, Double>> testExecs) {
-    Map<String, Map<String, Probability>> failcond = new HashMap<>();
+
+// Uses the execution map provided to produce Conditional probability that a test will pass based on other tests passing.
+private Map<String, Map<String, Probability>> buildCond(Map<String, Map<String, Double>> testExecs, boolean pass) {
```

Comment:
Instead of boolean pass, can you make it an enum of Pass and Fail? It's a little confusing this way